consul_exporter.go

```diff
@@ -202,7 +202,7 @@ func (e *Exporter) setMetrics(services <-chan []*consul_api.ServiceEntry) {
 
 			log.Infof("%v/%v status is %v", entry.Service.Service, entry.Node.Node, passing)
 
-			e.serviceNodesHealthy.WithLabelValues(entry.Service.Service, entry.Node.Node).Set(float64(passing))
```

Comment:
What Port are you trying to export here? A `ServiceEntry` struct does not have a port attribute https://godoc.org/github.com/hashicorp/consul/api#ServiceEntry.

I guess you mean Service.Port? It should be its own label instead of adding it to the node.